utils/vara-ui/src/components/button/helpers.ts

```diff
@@ -0,0 +1,8 @@
+const buttonSizes = ['x-small', 'small', 'medium', 'default', 'x-large'] as const;
+type IButtonSizes = (typeof buttonSizes)[number];
```

Comment:
We're not following `I` naming convention for types, and at the same time it's not an interface.

Also, I think separate file for small types like that is an overkill.